src/Symfony/Component/Validator/Context/ExecutionContext.php

```diff
@@ -107,7 +107,7 @@ public function __construct(
         private ValidatorInterface $validator,
         private mixed $root,
         private TranslatorInterface $translator,
-        private ?string $translationDomain = null,
+        private string|false|null $translationDomain = null,
```

Comment:
Don't we have to update the `addViolation()` method now to not call the translator when the translation domain is `false` (similar to what we have in `ConstraintViolationBuilder::addViolation()`?